pandas/core/window/rolling.py

```diff
@@ -1270,7 +1270,32 @@ def aggregate(self, func, *args, **kwargs):
         create_section_header("Returns"),
         template_returns,
         create_section_header("See Also"),
-        template_see_also[:-1],
+        template_see_also,
+        create_section_header("Examples"),
+        dedent(
+            """
+        >>> ser = pd.Series([0, 1, 5, 2, 8])
+
+        In order to get an instance of pandas.core.window.rolling.Window we need
+        to pass the parameter `win_type`.
+
+        >>> type(ser.rolling(2, win_type='gaussian'))
+        <class 'pandas.core.window.rolling.Window'>
+
+        In order to use the `Sci-py` gaussian window we need to provide the parameters
```

Comment:
I think we can just write "SciPy"

also, gaussian => Gaussian

Reply:
Done